content/kubernetes/re-clusters/multi-namespace.md

```diff
@@ -0,0 +1,176 @@
+---
+Title: Manage databases in multiple namespaces
+linktitle: Manage multiple namespaces
+description: Redis Enterprise for Kubernetes allows you to deploy to multiple namespaces within your Kubernetes cluster. This article shows you how to configure your Redis Enterprise cluster to connect to databases in multiple namespaces
+weight: 17
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/multi-namespace.md,
+    /kubernetes/re-databases/multi-namespace/,
+]
+
+---
+
+A single Redis Enterprise cluster (REC) can manage multiple Redis Enterprise database objects (REDB) in multiple namespaces within the same Kubernetes cluster.
```

Comment:
correctness suggestion:
"""
Multiple Redis Enterprise Database custom resources in multiple namespaces can be associated to a single Redis Enterprise Cluster custom resource. See [Manage databases in multiple namespaces]({{<relref "/kubernetes/re-clusters/multi-namespace.md">}}) for more information.
"""